{{cookiecutter.project_slug}}/.circleci/config.yml

```diff
@@ -146,22 +149,19 @@ jobs:
           paths:
             - vendor/bundle
 
-      - run:
-          name: Install CocoaPod Spec 
-          command: curl https://cocoapods-specs.circleci.com/fetch-cocoapods-repo-from-s3.sh | bash -s cf
-      - run:
-          name: Install Ruby and Fastlane
-          command: |
-           brew install ruby
-           gem install fastlane
```

Comment:
CocoaPod spec download is no longer necessary in 1.8, Ruby is already available in the system, bundle install already installs fastlane.